deployment/frontend/src/components/topics/TopicCard.tsx

```diff
@@ -24,13 +24,13 @@ export default function TopicCard({
                     }`}
                     alt={`Topic - ${topic.title}`}
                     fill
-                    className="object-cover"
+                    className="object-contain"
```

Comment:
@steveoni not sure about this - I thnk it should be cover?

Reply:
The other one caused stretching. Can you check it out locally @demenech